ambry-api/src/main/java/com.github.ambry/config/RouterConfig.java

```diff
@@ -262,8 +262,8 @@ public RouterConfig(VerifiableProperties verifiableProperties) {
     routerLatencyToleranceQuantile =
         verifiableProperties.getDoubleInRange("router.latency.tolerance.quantile", 0.9, 0.0, 1.0);
     routerBlobidCurrentVersion =
-        verifiableProperties.getShortFromAllowedValues("router.blobid.current.version", (short) 5,
-            new Short[]{1, 2, 3, 4, 5});
+        verifiableProperties.getShortFromAllowedValues("router.blobid.current.version", (short) 6,
```

Comment:
before we merge this, we should change the current version back to 5 to prevent any deployment hiccups

Reply:
Not necessary - the blob id version is expected to be overriden (and usually is)